app/commands/command/destroy/legislation.rb

```diff
@@ -0,0 +1,24 @@
+module Command
+  module Destroy
+    class Legislation
```

Comment:
This command defines what it means to destroy Legislation.

TODO: need to add tests for this